### DISCUSSION TITLE
Can't access Workspace settings in the self-hosted version. Is this intentional?

### DISCUSSION THREAD

**Discussion by icyphox** [outline/outline#7945] [Self Hosting] [answered]
The title. The docs seem to indicate this is possible and don't mention this being disabled for self-hosted. Am I missing something?

**Comment by tommoor** [comment]
It would probably mean that your user account is not an admin

**Reply by icyphox** [reply]
That’s weird, I’m the only user currently and I set up the whole thing. How do I make myself admin?

**Reply by tommoor** [reply]
Weird indeed, if you're in this state you'd have to update the `role` column in the database directly.

**Reply by icyphox** [reply]
Perhaps I’ll just clear out the DB completely for a full reset and start over. Do you have any instructions handy for that?

**Reply by tommoor** [reply, marked as answer]
If the db is empty migrations to create everything will run automatically when the server starts

**Reply by icyphox** [reply]
Oh neat, that did the trick. I deleted the Docker volumes and restarted all containers. Thanks for the assist!
